src/components/SwitchLanguage.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+    import { useI18n } from 'vue-i18n'
+    const { locale, t } = useI18n({ useScope: 'global' })
+
+    const langs = ['en', 'fr']
+</script>
+
+<template>
+    <div class="">
+        <span class="language">{{ t('Language') }}</span>
+        <select v-model="locale">
+            <option v-for="(lang, i) in langs" :key="`Lang${i}`">
```

Comment:
Vous pouvez utiliser la lang directement en key

```suggestion
            <option v-for="(lang, i) in langs" :key="lang">
```